src/components/ActionTile/ActionTile.module.scss

```diff
@@ -0,0 +1,105 @@
+@use '@assets/global.scss' as g;
+
+.icons {
+  width: 50px;
+  height: 50px;
+  border-radius: 50%;
+
+  display: flex;
+  text-align: center;
+  justify-content: center;
+  align-items: center;
+
+  @include g.md-breakpoint {
+    margin-top: 15px;
+  }
+}
+
+.tile {
```

Comment:
If this class isn't being used in the tsx, better practice to use a mixin (declare it with a % instead of a . and do @extends %tile)

Reply:
thanks, I'm now using mixins

Reply:
Lovely mixins